agenda/presentation/src/main/java/com/pronoidsoftware/agenda/presentation/detail/reminder/ReminderDetailScreen.kt

```diff
@@ -89,6 +90,25 @@ internal fun ReminderDetailScreen(
     val spacing = LocalSpacing.current
     val dividerColor = TaskyWhite2
 
+    if (state.isShowingCloseConfirmationDialog) {
+        val onCancelAction = when {
+            state.isEditingTitle -> ReminderDetailAction.OnCancelCloseTitle
+            state.isEditingDescription -> ReminderDetailAction.OnCancelCloseDescription
+            else -> ReminderDetailAction.OnCancelClose
+        }
+        val onConfirmAction = when {
+            state.isEditingTitle -> ReminderDetailAction.OnConfirmCloseTitle
+            state.isEditingDescription -> ReminderDetailAction.OnConfirmCloseDescription
+            else -> ReminderDetailAction.OnConfirmClose
+        }
+        TaskyDialog(
+            title = stringResource(id = R.string.close_dialog_title),
+            description = stringResource(id = R.string.confirm_close),
+            onCancel = { onAction(onCancelAction) },
+            onConfirm = { onAction(onConfirmAction) },
+        )
+    }
```

Comment:
Ah, it seems you implemented the edit text screen as a dialog, is there a particular reason or didn't this get clear from the mockups? 🤔

Reply:
Ah, no, the edit text screen is indeed a full screen composable. This handling here is for displaying the "Are you sure you want to close without saving?" dialog upon trying to go back. In my head this made sense for consolidating the actions for handling that dialog. But perhaps not if it led you to believe otherwise 😅

Reply:
Ah, then I misunderstood that. The confirm close title/confirm close description sounded like we're closing the edit screen here without saving. But then it's good 👍